modules/virt-linux-bridge-nad-port-isolation.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-linux-bridge-nad-port-isolation.adoc_{context}"]
+= Enabling port isolation for a Linux bridge NAD
+
+You can enable port isolation for a Linux bridge network attachment definition (NAD) so that virtual machines (VMs) or pods that run on the same virtual LAN (VLAN) can operate in isolation from one another. The Linux bridge NAD creates a virtual bridge, or _virtual switch_, between network interfaces and the physical network.
+
+Isolating ports in this way can provide enhanced security for VM workloads that run on the same node. 
+
+.Prerequisites
+
+* For VMs, you configured either a static or dynamic IP address for each VM. See "Configuring IP addresses for virtual machines".
+* You created a Linux bridge NAD by using either the web console or the command-line interface. 
+* You ensured that `spec.config.hairpinMode` in the Linux bridge NAD was not set to `true`.
```

Comment:
This surprised me. Where is this requirement coming from?

Do you know @ormergi ?

Reply:
I missed that line, my bad.
I dont know of such limitation, isolation should work alongside hairpin mode.
@dfitzmau  Could you please share a ref for that? If there is none, we should omit this line.

Reply:
I'm unable to find the exact source for this, so I removed the line. Thanks for confirming!